docs/source/models/cause.rst

```diff
@@ -4,66 +4,119 @@
 Modeling Causes
 ===============
 
-.. todo::
+.. note::
 
-   Overview of modeling GBD causes.
-
-.. contents:
-
-What is a cause?
-----------------
+   Vivarium has the infrastructure to model causes at any level in the GBD hierarchy.
+   However, it is recommended to model GBD causes at the most detailed level to avoid 
+   problems with missingness in the GBD cause hierarchy.
 
 The structure of a cause model
 ------------------------------
 
+**Cause entities:** Searching the GBD cause_id associated with the cause. Find out
+affected outcomes by this cause (either mortality, disability, or both). Start thinking 
+about the restrictions such as certain ages or sexes applied to this cause.
+(useful package: `gbd_mapping <https://vivarium.readthedocs.io/projects/gbd-mapping/
+en/latest/gbd_mapping.html>`_)
+
+**Modeling Strategy:** To address proper methods that used to model causes we interested.
```

Comment:
"we interested" should be "we are interested in" or "in which we are interested".